components/UnorderedList/src/index.scss

```diff
@@ -0,0 +1,17 @@
+@import "~@utrecht/components/unordered-list/bem";
+
+.denhaag-unordered-list {
+  @extend .utrecht-unordered-list;
+
+  color: var(--denhaag-unordered-list-color);
+  margin-inline-start: var(--denhaag-unordered-list-margin-inline-start);
+  padding-inline-start: var(--denhaag-unordered-list-padding-inline-start);
```

Comment:
`.utrecht-unordered-list` already has `padding-inline-start: var(--utrecht-unordered-list-padding-inline-start);`. We should remove this line and define that token instead.

Reply:
removed these css properties + values + tokens in commit https://github.com/nl-design-system/denhaag/pull/767/commits/0c90ebf68b98f007db7a83e19f33b890b53e0e91